cdc/owner/owner.go

```diff
@@ -174,7 +178,14 @@ func (o *Owner) Tick(stdCtx context.Context, rawState orchestrator.ReactorState)
 			cfReactor = o.newChangefeed(changefeedID, o.gcManager)
 			o.changefeeds[changefeedID] = cfReactor
 		}
+
+		startTime := time.Now()
 		cfReactor.Tick(ctx, changefeedState, state.Captures)
+		costTime := time.Since(startTime)
+		if costTime > changefeedLogsWarnDuration {
+			log.Warn("changefeed tick took too long", zap.String("changefeed", changefeedID), zap.Duration("duration", costTime))
+		}
+		changefeedReactorTickDuration.WithLabelValues(changefeedID).Observe(costTime.Seconds())
```

Comment:
We should avoid using `WithLabelValues` to create metric observer in hot path, which will cost too much CPU.
A better solution is to pre allocate prometheus labels with some reusable observer, such as what we use in processor

https://github.com/pingcap/tiflow/blob/99389d612d4c64bfa22080a98cf5a2ef16386aea/cdc/processor/processor.go#L240-L248